torch/types.py

```diff
@@ -22,6 +22,9 @@
 _size = Union[torch.Size, List[_int], Tuple[_int, ...]]
 _layout = torch.layout
 
+class SymInt:
```

Comment:
this is a temporary workaround. Eventually SymInt would be `Union[int, SymIntNode]` and we will be hitting the exact same error: 

```
error: Overloaded function signature 2 will never be matched: signature 1's parameter type(s) are the same or broader  [misc]
```